src/main/java/hudson/tasks/junit/JUnitTask.java

```diff
@@ -12,4 +12,6 @@
     boolean isKeepLongStdio();
 
     boolean isAllowEmptyResults();
+    
+    boolean isMakeUnstable();
```

Comment:
Breaks the binary compatibility of the interface.
In order to do so, you may need to update the plugin dependency to Jenkins Core 2.60.3 and Java 8. Then you will be able to add a default method to the interface

Reply:
Updated Java to 8. Jenkins core is 2.7.3 before.